### ISSUE TITLE
Bulk and/or enterprise licensing?

### ISSUE THREAD

**Issue by jamespfluger** [ForkIssues/Tracker#965]
Fork is becoming the most commonly used application in our team and (hopefully) soon the company. While I am willing to pay quadruple the price it is being sold for, some of my team members do not want to pay for a license since they will only use it at work.

Are there any plans to have the ability to purchase licenses in just a single enterprise license? I realize that enterprise licensing could require a new burden, and it would also change the product from an independent userbase to a independent/corporate one.

tl;dr - are there any plans for bulk licensing?

**Comment by DanPristupov** [comment]
Hi.

> While I am willing to pay quadruple the price it is being sold for

Ha-ha. Thank you :)

Regarding the volume purchase:

1. You can already buy multiple licenses here: https://fork.dev/buy
2. Just buy it on the company email and then share with the employee.
3. The licenses are transferable. You can deactivate the license on one computer and activate it for other employee.
4. You can buy additional licenses in the future if needed.

We also have company licenses but we need to discuss each case separately (support@fork.dev).

**Comment by jamespfluger** [comment]
Thanks for the information! The transferable licensing could be an issue with developers coming and going, which means we'd have to deactivate the licenses if they didn't use them.

I'll look into getting licenses for our team, thanks!
